app/views/sign_up/completions/show.html.slim

```diff
@@ -1,10 +1,10 @@
--title t("titles.loa3_verified.#{session[:sp][:loa3]}", app: APP_NAME)
+-title t("titles.loa3_verified.#{session['sp][:loa3']}", app: APP_NAME)
```

Comment:
Yep. I wouldn't spend too much time reviewing this since this is still a WIP. I used a regex search and replace in Sublime Text and will need to clean up these kinds of wrong replacements.